src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1557,12 +1557,16 @@ class C
                 Dim projection = projectionBufferFactory.CreateProjectionBuffer(Nothing, New Object() {otherExposedSpan, subjectBufferExposedSpan}.ToList(), ProjectionBufferOptions.None)
 
                 Dim view = textViewFactory.CreateTextView(projection)
```

Comment:
Maybe look for other calls to `EditorFactory.CreateView`.  I'm pretty sure there are a bunch of them.